scripts/SANS/sans/gui_logic/presenter/run_tab_presenter.py

```diff
@@ -282,14 +289,18 @@ def set_view(self, view):
         if view is not None:
             self._view = view
 
-            # Add a listener to the view
             listener = RunTabPresenter.ConcreteRunTabListener(self)
             self._view.add_listener(listener)
 
-            # Default gui setup
-            self._default_gui_setup()
+            self.default_gui_setup()
             self._view.disable_process_buttons()
 
+            for presenter in self._common_sub_presenters:
+                # TODO when the below presenters are converted to use a common presenter
```

Comment:
Did you forget to delete this comment?

Reply:
In this case, no since the above presenters don't use the common interface yet